v2/pkg/templates/types/types.go

```diff
@@ -22,19 +22,23 @@ const (
 	HeadlessProtocol
 	NetworkProtocol
 	WorkflowProtocol
+	SSLProtocol
```

Comment:
you could rename the package `protocols` and then you could leave out the `Protocol` suffix here

Reply:
There's already a protocol package hence the naming was chosen as types, as there's a list of protocol types